Database Files/1 table.sql

```diff
@@ -0,0 +1,33 @@
+use GreenBeretOpsDB
+GO
+
+drop table if exists Mission
+GO
+
+create table dbo.Mission(
+    MissionId int not null identity primary key,
+    FirstName varchar(35) not null constraint Mission_FirstName_cannot_be_blank check(FirstName > ''),
+    LastName varchar(35) not null constraint Mission_LastName_cannot_be_blank check(LastName > ''),
+    YearSoldierBorn int not null constraint Mission_YearSoldierBorn_must_be_between_1776_and_current_date check(YearSoldierBorn between 1776 and getdate()),
```

Comment:
GreenBerets started 1952.

Reply:
Where would I add a constraint for that?